float64[16] stop_table

